src/debugAdapterInterfaces.d.ts

```diff
@@ -179,7 +181,7 @@ export interface IDebugAdapter {
     shutdown(): void;
 
     initialize(args: DebugProtocol.InitializeRequestArguments, requestSeq?: number): PromiseOrNot<DebugProtocol.Capabilities>;
-    launch(args: ILaunchRequestArgs, requestSeq?: number): PromiseOrNot<void>;
+    launch(args: ILaunchRequestArgs, telemetryPropertyCollector: ITelemetryPropertyCollector, requestSeq?: number): PromiseOrNot<void>;
```

Comment:
After I get some consensus about this design. I can modify all the request methods as well.